cmd/kubeadm/app/cmd/version.go

```diff
@@ -22,8 +22,8 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"k8s.io/client-go/pkg/version"
```

Comment:
This is an ok transition for now, but it really seems like `kubeadm`'s version should be controlled by kubeadm, not whatever level of the client you vendored.